lib/references.js

```diff
@@ -7,10 +7,9 @@ var getPageLinks               = utils.getPageLinks
 var isInternalLink             = utils.isInternalLink
 var prepareInternalLink        = utils.prepareInternalLink
 
-
 var pages = []
```

Comment:
you're quite picky about code style on files you didn't look at for more than a year..